src/inpage-panel.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <title>Firefox Relay</title>
+    <link rel="stylesheet" href="/css/relay.css" />
+    <link rel="stylesheet" href = "/css/first-run.css" />
+    <script src="/js/libs/browser-polyfill.min.js"></script>
+    <script src="/js/shared/utils.js"></script>
+    <script src="/js/shared/metrics.js"></script>
+    <script src="/js/other-websites/fathom.js"></script>
+    <script src="/js/other-websites/email_detector.js"></script>
+    <script src="/js/other-websites/inpage_menu.js"></script>
+    <script src="/js/other-websites/fill_relay_address.js"></script>
```

Comment:
```suggestion
    <script src="/js/other-websites/inpage_menu.js"></script>
```

Reply:
Removed in ed515009a70c4fd3b452eff9955d37539efb41bc